context/app/static/js/stores/useEntityStore.ts

```diff
@@ -41,6 +43,8 @@ export const createEntityStore = ({ springs }: { springs: ReturnType<typeof useS
           summaryEntry: entry,
         },
       }),
+    summaryHeight: 0,
+    setSummaryHeight: (val: number) => set({ summaryHeight: val }),
```

Comment:
Non-blocker, just an idea:

```suggestion
    setSummaryHeight: (summaryHeight: number) => set({ summaryHeight }),
```